etc/docker/elyra/Dockerfile

```diff
@@ -39,9 +39,11 @@ RUN python -m pip install --upgrade pip && \
     npm install -g yarn && \
     npm install -g npm && \
     cd /tmp && git clone https://github.com/elyra-ai/elyra.git && \
-    cd /tmp/elyra && git checkout tags/v1.0.0rc3 -b v1.0.0rc3 && make install
+    cd /tmp/elyra && git checkout tags/v1.0.0rc2 -b v1.0.0rc2 && make install
```

Comment:
this should be rc3, you might need to update your master

Reply:
done